extensions/azurecore/src/azureResource/utils.ts

```diff
@@ -463,7 +461,6 @@ export async function makeHttpRequest<B>(
 	requestHeaders: Record<string, string> = {}
 ): Promise<AzureRestResponse<B>> {
 	const result: AzureRestResponse<B> = { response: undefined, errors: [] };
-	const httpClient: HttpClient = getProxyEnabledHttpClient();
```

Comment:
Finally clean this up as well?

Reply:
I also think this was referenced elsewhere but not actually used.. so please clean up.